modules/nf-core/orthofinder/tests/main.nf.test

```diff
@@ -52,7 +51,51 @@ nextflow_process {
             def stable_files = all_files.findAll { it.name in stable_file_names }
 
             assert snapshot(
-                all_file_names,
+                stable_files,
+                process.out.versions[0]
+            ).match()
+        }
+
+    }
+
+
+    test("sarscov2 - candidatus_portiera_aleyrodidarum - proteome - resume") {
+
+        when {
+            process {
+                """
+                file(params.modules_testdata_base_path + 'genomics/sarscov2/genome/proteome.fasta', checkIfExists: true)
+                    .copyTo("${workDir}/sarscov2.fasta")
+
+                def file_a = file("https://raw.githubusercontent.com/nf-core/test-datasets/proteinfold/testdata/sequences/H1065.fasta")
+                def file_c = file("s3://temporary-nfcore-module-data/test/WorkingDirectory")
```

Comment:
This line will change to :

def file_c = file("https://raw.githubusercontent.com/nf-core/test-datasets/modules/data/delete_me/orthofinder/WorkingDirectory")

ONce the PR is through on test datasets